wp-sw-cache/wp-sw-cache-main.php

```diff
@@ -2,6 +2,7 @@
 
 require_once(plugin_dir_path(__FILE__).'wp-sw-cache.php' );
 require_once(plugin_dir_path(__FILE__).'wp-sw-cache-db.php');
+require_once(plugin_dir_path(__FILE__).'vendor/mozilla/wp-sw-manager/class-wp-sw-manager.php');
```

Comment:
Wondering how we should handle this.  Add a submodule to this project and do the same with SVN when we upload it to WordPress.org?